diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index b6ba30425c8cb2e369903f7dbf5769654c5eeeaf..ff26702470372995d9a3db12c5a7789f1fd928dc 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2469,6 +2469,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
