diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 99e07e30f3182b78f9898df566f0a42f0ec0fe03..74a4d2df7911d16e7575bb2abb69b325d2f76884 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -324,7 +324,7 @@ void ToolbarView::Init() {
   } else {
     // TODO(crbug.com/932818): Remove this once the
     // |kAutofillEnableToolbarStatusChip| is fully launched.
-    avatar_ = AddChildView(std::make_unique<AvatarToolbarButton>(browser_));
+    avatar_ = AddChildView(std::make_unique<BraveAvatarToolbarButton>(browser_));
     avatar_->SetVisible(show_avatar_toolbar_button);
   }
 
