diff --git a/chrome/browser/ui/chrome_pages.cc b/chrome/browser/ui/chrome_pages.cc
index 5b3ee8b60b717b762e702bb813f3ebf3ba4a9338..12a7e8ae887f061dc2bb32f9dd841bac2917b5ed 100644
--- a/chrome/browser/ui/chrome_pages.cc
+++ b/chrome/browser/ui/chrome_pages.cc
@@ -307,7 +307,7 @@ void ShowSlow(Browser* browser) {
 #endif
 }
 
-GURL GetSettingsUrl(const std::string& sub_page) {
+BRAVE_GET_SETTINGS_URL
   return GURL(std::string(kChromeUISettingsURL) + sub_page);
 }
 
