diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index 6960ec08c57fa50b6f626b07e684702f00d9bf7e..8acb8afe7020afe91ded15e7db493089cb6bd474 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -2,6 +2,8 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/permissions/sources.gni")
+
 source_set("permissions_common") {
   sources = [
     "features.cc",
@@ -92,6 +94,7 @@ source_set("permissions") {
     "//ui/base",
     "//url",
   ]
+  sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
