diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index eb31d15e8449484010181d5a2b1dce6b18f0067c..036b8d815312795394eb9c9e0f5bdb11b3f1a086 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1831,6 +1831,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
