diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index 04791dc26a07896c39c3482632936e442335d49e..ff00640bb2c70e3ae69a851b694b5f4a55903afe 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -47,6 +47,9 @@
   "chrome/app/chromium_strings.grd": {
     "messages": [400],
   },
+  "chrome/app/brave_strings.grd": {
+    "messages": [400],
+  },
   "chrome/app/google_chrome_strings.grd": {
     "messages": [400],
   },
@@ -437,6 +440,9 @@
     "META": {"align": 100},
     "messages": [2700],
   },
+  "components/components_brave_strings.grd": {
+    "messages": [2700],
+  },
   "components/components_google_chrome_strings.grd": {
     "messages": [2700],
   },
