diff --git a/ui/native_theme/native_theme_mac.mm b/ui/native_theme/native_theme_mac.mm
index 1c56f4f45addc85a524538d06aa17bbbf513b95b..cba4efe589f33cecfc32d77615065e70bcefa824 100644
--- a/ui/native_theme/native_theme_mac.mm
+++ b/ui/native_theme/native_theme_mac.mm
@@ -176,6 +176,7 @@ base::Optional<SkColor> NativeThemeMac::GetOSColor(
 
   // Even with --secondary-ui-md, menus use the platform colors and styling, and
   // Mac has a couple of specific color overrides, documented below.
+  GET_BRAVE_COLOR(color_id)
   switch (color_id) {
     case kColorId_EnabledMenuItemForegroundColor:
       return skia::NSSystemColorToSkColor([NSColor controlTextColor]);
